data.json

```diff
@@ -52,13 +61,17 @@
   {
     "name": "new_york",
     "url": "http://gis.ny.gov/gisdata/data/ds_921/SAM_Master_Statewide.gdb.zip",
-    "hash": "9a52775995af73fe5e6cd1aeaa9b52fbe9c35389c8657dbd75f99149caa10fd9",
+    "hash": "dda07a26bb25b5c900d0d1464ade072e3aa769372bf4d9bca75fa05ddae96122",
     "file": "SAM_Master_Statewide_Database.gdb",
     "count": 4761260,
     "fields": {
-      "Address": {
-        "type": "static",
-        "value": "AddressLabel"
+      "Number": {
+        "type": "multi",
+        "value": ["PrefixAddressNumber", "AddressNumber", "SuffixAddressNumber"]
+      },
+      "Street": {
+        "type": "multi",
+        "value": ["PreModifier", "PreDirectional", "PreType", "StreetName", "PostType", "PostDirectional", "PostModifier"]
```

Comment:
Here's where it gets really interesting.  The usaddress parser has both!  The following are in usaddress's [`labeled.xml`](https://github.com/datamade/usaddress/blob/master/training/labeled.xml) training file:

**[Exhibit A: `PreModifier` -> `PreDirectional`](https://github.com/datamade/usaddress/blob/master/training/labeled.xml#L972)**

    5442 OLD WEST STATE ROUTE 21,IMPERIAL, MO 63052

parses to:

```xml
<AddressString><AddressNumber>5442</AddressNumber> <StreetNamePreModifier>OLD</StreetNamePreModifier> <StreetNamePreDirectional>WEST</StreetNamePreDirectional> <StreetNamePreType>STATE</StreetNamePreType> <StreetNamePreType>ROUTE</StreetNamePreType> <StreetName>21,</StreetName> <PlaceName>IMPERIAL,</PlaceName> <StateName>MO</StateName> <ZipCode>63052</ZipCode></AddressString>
```

**[Exhibit B: `PreDirectional` -> `PreModifier`](https://github.com/datamade/usaddress/blob/master/training/labeled.xml#L859)**

    135 W OLD US HIGHWAY 90

parses to:

```xml
<AddressString><AddressNumber>135</AddressNumber> <StreetNamePreDirectional>W</StreetNamePreDirectional> <StreetNamePreModifier>OLD</StreetNamePreModifier> <StreetNamePreType>US</StreetNamePreType> <StreetNamePreType>HIGHWAY</StreetNamePreType> <StreetName>90</StreetName></AddressString>
```

So, for now, I think its safe to just keep things as they are, and the parser will always just return the values as they occur in the address string.